apps/fluent-tester/src/TestComponents/CheckboxExperimental/CheckboxTest.tsx

```diff
@@ -54,25 +46,27 @@ const SizeCheckbox: React.FunctionComponent = () => {
 };
 
 const OtherCheckbox: React.FunctionComponent = () => {
-  const [isCheckedControlled1, setCheckedControlled1] = React.useState(false);
-  const onChangeControlled1 = React.useCallback((checked) => {
-    setCheckedControlled1(checked);
+  const [isChecked, setisChecked] = React.useState(false);
+
+  const setCheckedTrue = React.useCallback(() => {
+    setisChecked(true);
   }, []);
 
-  const [isCheckedControlled2, setCheckedControlled2] = React.useState(true);
-  const onChangeControlled2 = React.useCallback((checked) => {
-    setCheckedControlled2(checked);
+  const setCheckedFalse = React.useCallback(() => {
+    setisChecked(false);
   }, []);
 
   return (
-    <View>
-      <Checkbox label="This is a controlled Checkbox" onChange={onChangeControlled1} checked={Boolean(isCheckedControlled1)} />
-      <Checkbox
-        label="Checkbox rendered with labelPosition 'before' (controlled)"
-        onChange={onChangeControlled2}
-        labelPosition="before"
-        checked={Boolean(isCheckedControlled2)}
-      />
+    <View style={commonStyles.customisedChecked}>
+      <Button onClick={setCheckedTrue} size="small">
+        Turn Checkbox True
```

Comment:
super nit: "Check controlled checkboxes below" and "Uncheck controlled checkboxes below" may be better wording, turning a checkbox true/false doesn't make as much sense, although you can probably figure out what it will do.

Reply:
yes , changed.

Reply:
Changes completed and approved , hence resolving.